octoprint_matrix_notifier/matrix.py

```diff
@@ -78,10 +79,11 @@ def upload_media(self, media_data, content_type):
         )
 
     def room_send_markdown_message(self, room_id, text):
+        html = markdown.markdown(text, extensions=['nl2br'])
         content = {
             "msgtype": "m.text",
-            "body": text,
+            "body": BeautifulSoup(html, 'html.parser').get_text(),
```

Comment:
What's the objective here? It's reasonably common in the matrix ecosystem to put markdown in the plain text body no?

Reply:
the plain text is used for notification in the APP and desktop.
and the notification can not render Markdown.
I personalty can read it fin, but if the notification get pushed small devices like a smart watch then most of the space is lost to Markdown format.